src/app/api/composio/connect/disconnect/route.ts

```
@@ -0,0 +1,84 @@
+    if (!connectionToVerify || connectionToVerify.entity_id !== userId) {
+        console.warn(`User ${userId} attempt to disconnect account ${connectedAccountId} not owned by them or not found.`);
+        // Do not explicitly say "not found" or "not owned" to avoid information leakage.
+        return NextResponse.json({ error: "Permission denied or connection not found." }, { status: 403 });
+    }
+
+
+    // Attempt to delete the connection
+    // Assuming a method like `delete` or `remove` exists on `connected_accounts` or `connectedAccounts`
+    let deleteSuccess = false;
+    try {
+      if (typeof (toolset.connected_accounts as any)?.delete === 'function') {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)